egs/librispeech/s5/local/chain/run_tdnn_6z.sh

```diff
@@ -1,7 +1,7 @@
 set -e
 
 # configs for 'chain'
-# this script is based on swbd's 6z script
+# this script is based on swbd's 6z script, but use left-biphone model
```

Comment:
Wasn't 7b which has larger number of parameters than 6z the best model ? Are you using the same number of parameters as 6z or 7b ?